projects/ad5758_sdz/Makefile

```diff
@@ -0,0 +1,21 @@
+####################################################################################
```

Comment:
Can you regenerate the Makefiles against master?

Reply:
Done.